selenium-tests/tests/all-alerts-test.js

```diff
@@ -0,0 +1,40 @@
+const { By, Key, Builder, until } = require("selenium-webdriver");
+const assert = require("assert");
+const chrome = require("selenium-webdriver/chrome");
+let driver;
+const chromeOptions = new chrome.Options();
+
+chromeOptions.addArguments("--headless");
+
+async function testAllAlertsPage() {
+  try {
+    driver = await new Builder()
+      .forBrowser("chrome")
+      .setChromeOptions(chromeOptions)
+      .build();
+
+    await driver.get("http://localhost/all-alerts.html");
+
+    // Test for presence of alerting header
+    let alertingHeader = await driver.findElement(By.css(".myOrg-heading")).getText();
+    assert.equal(alertingHeader, "Alerting", "Alerting header text is not correct");
+
+    // Test for presence of "Add Alert Rule" button
+    let addAlertButton = await driver.findElement(By.id("new-alert-rule"));
+    let addAlertButtonText = await addAlertButton.getText();
+    assert.equal(addAlertButtonText.includes("Add Alert Rule"), true, '"Add Alert Rule" button text is not correct');
+
```

Comment:
Can you also add checks: 
1. if the alert button is clickable.
2. once you click the `Add Alert Rule` button, that a form is displayed.
3. test if all the form attributes are present.
4. test that the table is populated once you add the alert rule.